Snowflake.Data.Tests/UnitTests/ConnectionPoolManagerSwitchTest.cs

```diff
@@ -0,0 +1,26 @@
+using NUnit.Framework;
+using Snowflake.Data.Client;
+using Snowflake.Data.Core.Session;
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    public class ConnectionPoolManagerSwitchTest
+    {
+        private readonly string _connectionString1 = "database=D1;warehouse=W1;account=A1;user=U1;password=P1;role=R1;";
+        private readonly string _connectionString2 = "database=D2;warehouse=W2;account=A2;user=U2;password=P2;role=R2;";
+
+        [Test]
+        public void TestRevertPoolToPreviousVersion()
+        {
+            // act
+            SnowflakeDbConnectionPool.SetOldConnectionPoolVersion();
```

Comment:
As we test here SnowflakeDbConnectionPool, maybe the class should be called SnowflakeDbConnectionPoolTest?
And later we could add here other unit tests of SnowflakeDbConnectionPool.
Just an idea.